build/Dockerfile

```diff
@@ -12,8 +12,21 @@ RUN apt-get update \
 ############################################# Base image for Alpine #############################################
 FROM nginx:1.19.9-alpine AS alpine
 
-RUN apk add --no-cache libcap \
-	&& rm -rf /var/cache/apk/*
+RUN apk add --no-cache libcap
+
+
+############################################# Base image for Alpine with NGINX Plus #############################################
+FROM alpine:3.13 as alpine-plus
+ENV NGINX_PLUS_VERSION 24-r1
```

Comment:
currently, the other Plus images all have R23. Could we have one version of N+ across all images?

Reply:
3.13 is only supported in R24.

Reply:
would it make sense to upgrade other images to R24 first then?

Reply:
Created #1604 to update to R24

Reply:
@lucacome will we able to not use a specific patch version of N+ for Apline similarly to Debian in #1604?

Reply:
yep I'll update this PR once the other one is done